x-pack/plugins/lens/public/pie_visualization/render_helpers.test.ts

```diff
@@ -36,7 +36,7 @@ describe('render helpers', () => {
       ).toEqual(-100);
     });
 
-    it('returns epsilon when metric is 0 without fallback', () => {
+    it('returns 0 when metric is 0', () => {
```

Comment:
If decided so, it could perhaps be tested with negatives and maybe non-finite numbers (optional)